plone/app/contenttypes/behaviors/leadimage.pt

```diff
@@ -1,14 +1,14 @@
 <div class="leadImage"
      tal:condition="view/available"
      tal:define="scale_func context/@@images;
-                 scaled_image python: getattr(context.aq_explicit, 'image', False) and scale_func.scale('image', scale='mini')">
+                 scaled_image python: getattr(context.aq_explicit, 'image', False) and scale_func.scale('image', scale=view.scale_name)">
 
-  <figure class="newsImageContainer" tal:condition="python: scaled_image">
+  <figure tal:attributes="class python:view.css_class+'Container'" tal:condition="python: scaled_image">
```

Comment:
newsImageContainer does have some css styles applied. Wouldn't it be better to change it to some more generic name, like ``leadImageContainer``?